src/plugins/auto/plugin.cpp

```diff
@@ -369,6 +369,16 @@ IExecutableNetworkInternal::Ptr MultiDeviceInferencePlugin::LoadNetworkImpl(cons
             loadConfig.set_property(ov::hint::performance_mode(ov::hint::PerformanceMode::THROUGHPUT));
         }
     }
+    // Add core properties
+    auto config = inputConfig;
+    auto cache_dir = GetCore()->GetConfig(GetName(), ov::cache_dir.name());
+    if (cache_dir != ov::Any("")) {
+        config[ov::cache_dir.name()] = cache_dir.as<std::string>();
+    }
+    config[ov::auto_batch_timeout.name()] =
+        GetCore()->GetConfig(GetName(), ov::auto_batch_timeout.name()).as<std::string>();
+    config[ov::hint::allow_auto_batching.name()] =
+        GetCore()->GetConfig(GetName(), ov::hint::allow_auto_batching.name()).as<std::string>();
```

Comment:
why do we need to store these values? I supposed the idea is that AUTO_BATCH will query these values explicitly from Core if they are not passed by users explicitly. 

So, MultiDevice / AUTO should bypass only for cases:
```c++
core.compile_model(model, "AUTO", timeout(1000), perf_mode(tput)); // AUTO should redirect the value to BATCH plugin
```

Reply:
yes, the logic is it.
Here code is just to keep original AUTO's internal config manage flow.